### ISSUE TITLE
Restore font sizes etc after plotting

### ISSUE THREAD

**Issue by tlml** [serpentine-h2020/SEPpy#5]
**Issue**: 
Currently the plotting routines change matplotlib rcParams, which affects the users' plots after the SEPpy plotting has been used. 

**Potential solution**:
There are two ways to tackle this. One is to use `ax.title.set_fontsize()` etc to set the fonts for the individual image. The other is to store and restore the rcParams before and after plotting in SEPpy:

```
  import matplotlib.pyplot as plt
  from matplotlib import rcParams
  
  # User has changed font size to 20
  rcParams.update({'font.size': 20})
  
  # "Script" stores  user's parameters, and changes font size to 20
  orig_rcparams = rcParams.copy()
  rcParams.update({'font.size': 30})

  fig, ax = plt.subplots()
  ax.plot(list(range(4)), list(range(4)))
  
  # "Script" restores user's parameters
  rcParams.update(orig_rcparams) 
  
  # User plots with their original rcParams
  fig, ax = plt.subplots()
  ax.plot(list(range(4)), list(range(4)))

```

**Comment by jgieseler** [comment]
I like your second approach! :+1:

**Comment by jgieseler** [comment]
Fixed with https://github.com/serpentine-h2020/SEPpy/pull/7, except for one little thing: As far as I understand @Christian-Palmroos' comment, for `tsa_plot()` the `rcParams` are not restored because that function runs interactively and doesn't come to a clearly defined end. Fortunately it doesn't change to much options, only these two:
``` python
plt.rcParams['axes.linewidth'] = 1.5
plt.rcParams['font.size'] = 12
```
And due to its nature, I think the `tsa_plot` function probably will not too often be run in a larger Notebook or analysis, where it would mess up the standard plotting parameters. Also, @Christian-Palmroos mentioned:
> For now I'll attach the original prcparams to a class attribute, so that the user may manually return the parameters after they are done with tsa.

**Comment by tlml** [comment]
Great! Interactivity can be a bit tricky, the class attribute is a good idea! Not sure if it would be a good idea to warn the user about it... Anyway, things break occasionally, I've noticed that switching between %matplotlib inline and %matplotlib widget also breaks things... But thanks!, restoring the rcParams will make things easier particularly for new users!

**Comment by jgieseler** [comment]
[Released in version 0.1.4](https://github.com/serpentine-h2020/SEPpy/releases/tag/v0.1.4) ([PyPI](https://pypi.org/project/seppy/0.1.4/))
